common/numbers.ts

```diff
@@ -34,6 +34,10 @@ export class Address extends Bytes {
   static fromString(s: string): Address {
     return changetype<Address>(typeConversion.stringToH160(s))
   }
+
+  static zero(): Address {
```

Comment:
Hmm it's strange to instantiate a 0 length address, since I believe `Address` is supposed to differentiate from `Bytes` in that it is always 20 bytes long. If this returned a zeroed out 20 byte address, it would make more sense.

Reply:
Agreed, I'll fix it.